Traceback (most recent call last):
  File "C:\Users\r.herron\Anaconda3\envs\fina-6333\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\r.herron\Anaconda3\envs\fina-6333\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\r.herron\Anaconda3\envs\fina-6333\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\r.herron\Anaconda3\envs\fina-6333\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\r.herron\Anaconda3\envs\fina-6333\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\r.herron\Anaconda3\envs\fina-6333\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\r.herron\Anaconda3\envs\fina-6333\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\r.herron\Anaconda3\envs\fina-6333\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
returns_a.loc[counts_a >= 250 ].mul(100).plot(kind='bar')
plt.ylabel('Total Return (%)')
plt.title('Intraday and Overnight Return Decomposition\nCalendar Year Total Returns')
plt.show()
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mValueError[0m                                Traceback (most recent call last)
[1;32m~\AppData\Local\Temp\ipykernel_7812\3097142454.py[0m in [0;36m<module>[1;34m[0m
[1;32m----> 1[1;33m [0mreturns_a[0m[1;33m.[0m[0mloc[0m[1;33m[[0m[0mcounts_a[0m [1;33m>=[0m [1;36m250[0m [1;33m][0m[1;33m.[0m[0mmul[0m[1;33m([0m[1;36m100[0m[1;33m)[0m[1;33m.[0m[0mplot[0m[1;33m([0m[0mkind[0m[1;33m=[0m[1;34m'bar'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      2[0m [0mplt[0m[1;33m.[0m[0mylabel[0m[1;33m([0m[1;34m'Total Return (%)'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m      3[0m [0mplt[0m[1;33m.[0m[0mtitle[0m[1;33m([0m[1;34m'Intraday and Overnight Return Decomposition\nCalendar Year Total Returns'[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m      4[0m [0mplt[0m[1;33m.[0m[0mshow[0m[1;33m([0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\Anaconda3\envs\fina-6333\lib\site-packages\pandas\core\indexing.py[0m in [0;36m__getitem__[1;34m(self, key)[0m
[0;32m    965[0m [1;33m[0m[0m
[0;32m    966[0m             [0mmaybe_callable[0m [1;33m=[0m [0mcom[0m[1;33m.[0m[0mapply_if_callable[0m[1;33m([0m[0mkey[0m[1;33m,[0m [0mself[0m[1;33m.[0m[0mobj[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[1;32m--> 967[1;33m             [1;32mreturn[0m [0mself[0m[1;33m.[0m[0m_getitem_axis[0m[1;33m([0m[0mmaybe_callable[0m[1;33m,[0m [0maxis[0m[1;33m=[0m[0maxis[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    968[0m [1;33m[0m[0m
[0;32m    969[0m     [1;32mdef[0m [0m_is_scalar_access[0m[1;33m([0m[0mself[0m[1;33m,[0m [0mkey[0m[1;33m:[0m [0mtuple[0m[1;33m)[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\Anaconda3\envs\fina-6333\lib\site-packages\pandas\core\indexing.py[0m in [0;36m_getitem_axis[1;34m(self, key, axis)[0m
[0;32m   1190[0m [1;33m[0m[0m
[0;32m   1191[0m                 [1;32mif[0m [0mhasattr[0m[1;33m([0m[0mkey[0m[1;33m,[0m [1;34m"ndim"[0m[1;33m)[0m [1;32mand[0m [0mkey[0m[1;33m.[0m[0mndim[0m [1;33m>[0m [1;36m1[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[1;32m-> 1192[1;33m                     [1;32mraise[0m [0mValueError[0m[1;33m([0m[1;34m"Cannot index with multidimensional key"[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m   1193[0m [1;33m[0m[0m
[0;32m   1194[0m                 [1;32mreturn[0m [0mself[0m[1;33m.[0m[0m_getitem_iterable[0m[1;33m([0m[0mkey[0m[1;33m,[0m [0maxis[0m[1;33m=[0m[0maxis[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;31mValueError[0m: Cannot index with multidimensional key
ValueError: Cannot index with multidimensional key

